piper/buttondialog.py

```diff
@@ -71,17 +71,21 @@ class ButtonDialog(Gtk.Dialog):
     search_entry = GtkTemplate.Child()
     search_bar = GtkTemplate.Child()
 
-    def __init__(self, ratbagd_button, buttons, *args, **kwargs):
+    def __init__(self, ratbagd_button, buttons, device_type, *args, **kwargs):
         """Instantiates a new ButtonDialog.
 
         @param ratbagd_button The button to configure, as ratbagd.RatbagdButton.
+
         @param buttons The buttons on this device, as [ratbagd.RatbagdButton].
+
+        @param devicetype The type of this device, as ratbagd.RatbagDeviceType.
```

Comment:
Added spaces

Reply:
i added them because vscode will squish the entire doc comment into one line with just one newline